LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -1699,7 +1701,7 @@ private static void GetIteConfigurationsB(ISuperIO superIO, Manufacturer manufac
                         v.Add(new Voltage("+3V Standby", 7, 10, 10));
                         v.Add(new Voltage("VBat", 8, 10, 10));
                         t.Add(new Temperature("System #1", 0));
-                        t.Add(new Temperature("Chipset", 1));
+                        t.Add(new Temperature("System #2", 1));
```

Comment:
While it is true that `System #2` temperature probe is under the chipset, this isn't actually the chipset probe itself. That is where the currently named PCH probe is located. It might be more ideal to rename PCH to Chipset. I was thinking for `System #2`, it could just be named `System (Under Chipset)` or something similar so it's at least easier to identify instead of a generic `System #` value. As for `System #1`, that is located under the PCIe x1 slot for all three boards. It could just be named `System (Under PCIe x1)` or something similar.